src/msw/textctrl.cpp

```diff
@@ -80,6 +80,12 @@
     #define CFE_AUTOBACKCOLOR 0x04000000
 #endif
 
+// missing defines for MinGW build
+#define	CFM_UNDERLINETYPE			0x00800000		// Many displayed by 3.0 
```

Comment:
You should surround these with `#ifndef`, like the one above. Also I think you can remove the comment, it is not useful here.

Reply:
@MaartenBent ,
As I don't know whether they all defined at the same time I will wrap each one of them in its own #ifndef. That should hopefully silence any MinGW version. Let me know if this is not the preferred way.